odk1-src/security-privacy.rst

```diff
@@ -63,7 +67,7 @@ Google Play Store
 
 Downloads from the Google Play Store are compiled into aggregated usage statistics.
 
-Crash reports you elect to send are provided to us as anonymous crash reports. By design, these do not contain survey field values or other device- or user- specific data.
+Crash reports you elect to send are provided to us as anonymous crash reports. By design, these do not contain device or user specific data.
```

Comment:
"as anonymous crash reports" -> anonymously

> these do not contain device or user specific data.

I don't think this is so accurate. Sometimes crash reports include network information which could be identifying. I also don't know what "device-specific data" is. We do see what devices get what crash. That could potentially be identifying.

Reply:
Google Play Store reports are anonymized by default. It's Firebase reports that are not.